data-sources/k6-prometheus/config.yml

```diff
@@ -0,0 +1,9 @@
+id: k6-prometheus
+displayName: k6 Prometheus
+description: |
+  New Relic can receive your k6 telemetry data via New Relic's Prometheus remote write integration. Within New Relic you can find your k6 performance data alongside your real users data and server side performance.
+install:
+  primary:
+    link:
+      url: https://k6.io/docs/results-output/real-time/prometheus-remote-write/
+icon: logo.svg
```

Comment:
It appears that the logo is actually a png, not an svg.

Reply:
changed to svg

Reply:
updated